app/src/main/java/org/simple/clinic/util/messagesender/SmsMessageSender.kt

```diff
@@ -0,0 +1,19 @@
+package org.simple.clinic.util.messagesender
+
+import android.content.Intent
+import android.net.Uri
+import androidx.appcompat.app.AppCompatActivity
+import javax.inject.Inject
+
+class SmsMessageSender @Inject constructor(
+    private val activity: AppCompatActivity
+) : MessageSender {
+
+  override fun send(phoneNumber: String, message: String) {
+    val uri = Uri.parse("smsto:$phoneNumber")
```

Comment:
Is this the official way to do this? AFAIK, there is a dedicated Intent provided by the platform instead of a URI scheme, right?

Reply:
Yes, this seems to be [official](https://developer.android.com/guide/components/intents-common#Messaging) way.